docs/conf/portland/2023/news/writing-day-announcement.rst

```diff
@@ -0,0 +1,98 @@
+:template: {{year}}/generic.html
+
+.. post:: Mar 21, 2023
+   :tags: {{year}}, portland-{{year}}, schedule
+
+===============================  
+Write the Docs <3's open source
+===============================
+   
+Dear open source friends,
+
+Send us your developer advocates, community managers, and projects. Tell them of Writing Day 
+and invite them to join us!
+
+In case this is the first time you're hearing about the Write the Docs conference or you're 
+unfamiliar with Writing Day, we got you:
+
+`Write the Docs <https://www.writethedocs.org/conf/portland/2023/>`_
+    the conference focused on all things related to software documentation
+
+`Writing Day <https://www.writethedocs.org/conf/portland/{{year}}/writing-day/>`_
+   is modeled after the concept of “code sprints” or “hackathons”, which are common in open-source conferences -- getting a bunch of 
+   people together and have them work towards a shared goal, such as creating or improving documentation.
+
+The Write the Docs conference has many enthusiastic attendees. These attendees, documentarians, want to learn new 
+technologies, sharpen their skills, and add to their portfolios. The world of open source has a seemingly endless 
+number of wonderful open source projects. Documentarians want to write the docs, your open source project docs. 
+
+It's a match made at Write the Docs!
+
+But wait, there's more! This year we have created swag bags, filled with one of a kind items. We'll be gifting them 
```

Comment:
I missed where we said we'd do this :-p

Reply:
@ericholscher suggested this at the last meeting. I think he wants to empty a closet 😂

Reply:
Heh yea -- did we drop the % discount from this post?